namespace-provisioner/reference.hbs.md

```diff
@@ -1,29 +1,7 @@
 # Namespace Provisioner reference
 
-This topic contains know issues and limitations
-## Known limitations and issues
+This topic contains know issues and limitations, the default resources and some technical details
```

Comment:
`typo`: Known Issue

Reply:
`and some technical details` is too generic

Reply:
Rephrase the paragraph